packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -91,8 +91,7 @@ const createConceptDisplays = (compareConceptDisplays) => {
   return null;
 };
 
-const idForDatetimeDisplays = `${uuidv4()}-datetimeDisplay`;
-const createDatetimeDisplays = (compareDatetimeDisplays) => {
+const createDatetimeDisplays = (compareDatetimeDisplays, idForDatetimeDisplays) => {
```

Comment:
This change was made so that the uuid is the same inside the ClinicalResultBloodPressure component and the createDatetimeDisplay method without hardcoding it. The hard-coded uuid was different for those 2 instances when spyOn uuuid